### PR TITLE
CRAYSAT-1417: Add docs-sat RPM into product stream

### PR DESCRIPTION
**Pull request by ekamin-hpe**
* Add DOCS_SAT_VERSION to component_versions.sh
* Add code (copied from csm/release.sh) to extract the docs-sat
  RPM into a directory within the release distribution.
* Remove the code that flattens the repository directory, because
  now we are including both x86_64 and noarch packages.
* Add `rpm2cpio` to list of requirements in release.sh

Test Description:
* I built this branch of the release distribution
* After building the release distribution, I manually inspected
  the docs/ directory.
* I installed the release distribution on an internal system, and
  manually inspected the RPM repositories that were created in
  Nexus.
* I verified that the docs-sat RPM was installed, as well as the
  expected version of cray-sat-podman.

## Pull Request Checklist

- [X] Version number(s) incremented, if applicable
- [X] Copyrights updated
- [X] License file intact
- [X] Target branch correct
- [X] CHANGELOG.md updated
- [X] Testing is appropriate and complete, if applicable
- [X] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: component_versions.sh
Comment (ekamin-hpe) [inline component_versions.sh:24]: I think for this what we'll have to do is wait until the other pull requests merge, and then possibly to a merge to main and then update this file to include proper main branch versions. We could also potentially use the integration branch versions.

Comment (haasken-hpe) [reply component_versions.sh:24]: I agree that we should merge the other PRs first and probably do an integration -> main merge and then update this.

**Review by haasken-hpe — APPROVED** [review]

**Review by jack-stanek-hpe — APPROVED** [review]

**Pull request merged**
